functional-tests/Readme.md

```diff
@@ -48,28 +102,49 @@ sourceSets {
    }
 }
 ```
-## Questions and issues
 
-Please ask questions on our [Slack Channel][slack_channel] and raise issues in [BDDStack issue tracker][issue_tracker].
+# Troubleshooting Guide
```

Comment:
So far this section only has 1 troubleshooting bullet.  But the idea is to add all of the weird *gotchas* that exist, and hopefully save someone time and effort in future.